go/libkb/errors.go

```diff
@@ -2131,6 +2131,17 @@ func (e ChatClientError) IsImmediateFail() (chat1.OutboxErrorType, bool) {
 
 //=============================================================================
 
+type ChatUsersAlreadyInConversationError struct {
+	Uids   []keybase1.UID
+	ConvID chat1.ConversationID
```

Comment:
what is this used for?

Reply:
I'll drop it. thought it would be useful for logging/debug purposes as it.